src/cortex_cli/cortex_cli.py

```diff
@@ -322,6 +323,10 @@ def logout(config_file: str, keep_tokens: str, force: bool) -> None:
     base_url, realm, client_id = config['base_url'], config['realm'], config['client_id']
     tokens_file = config['tokens_file']
 
+    if not Path(tokens_file).is_file():
+        click.echo('Not logged in.')
```

Comment:
The ticket said "Not logged in. Run 'cortex auth login' to log in." But I guess prompting the user to log in does not make sense if he wants to log out.

Reply:
Fixed the ticket expectations.